nym-vpn-desktop/src-tauri/src/commands/connection.rs

```diff
@@ -248,7 +248,7 @@ pub async fn disconnect(
     })?;
     debug!("Stop message sent");
 
-    Ok(app_state.state)
+    Ok(app_state.state.clone())
```

Comment:
why these `clone()` here?

Reply:
It's because the connection state now contains a String with the connected gateway, so it does not implement Copy anymore